website/docs/apps/autoplot_app/README.md

```diff
@@ -99,32 +100,34 @@ Tailscale network.
 You can control your Amiga from a web browser at any time, even when the Autoplot app is not active
 on the Brain display.
 
-#### Remote Keyboard Control
+## How to Control your Amiga
 
-Experience direct control of your Amiga robot from any web browser.
-Similar to video game controls, this feature allows you to command the robot using simple keyboard inputs.
+You can control your Amiga robot in two ways:
 
-- **W**: Propels the robot forward
-- **A**: Rotates the robot counter-clockwise on the spot
-- **D**: Rotates the robot clockwise on the spot
+**Manual Control**: Use the [pendant](/docs/pendant),
+the [kartech](https://kar-tech.com/single-axis-joystick-universal-mega-system.html)
+or [dashboard](/docs/dashboard/dashboard-user-guide)
+while physically present with the robot, ideal for teaching the robot new tracks.
 
-These commands engage the robot's autonomy systems to perform the actions safely and precisely,
-setting incremental goals for the robot to achieve.
+Here are instructions for using the pendant and the kartech:
 
-:::info Autonomy Required for Remote Control
-Remote Keyboard Control requires the robot to be in `automode`.
-This mode ensures that all movements are managed safely by the robot's autonomous systems.
-:::
+![pendant](https://github.com/farm-ng/amiga-dev-kit/assets/39603677/4a8fbe60-7780-4f32-b92f-6ce895a706e0)
+![kartech](https://github.com/farm-ng/amiga-dev-kit/assets/39603677/99aee18e-e7d2-41c5-888b-6d38365c699e)
 
-## Manual vs. Remote Control
+**Remote Control**: From your web browser, you can remotely operate the robot without needing
+to be in the same physical location.
+For that, you can use a keyboard for driving your Amiga, or use an [8Bitdo
+Sn30](https://shop.8bitdo.com/products/8bitdo-sn30-pro-bluetooth-gamepad) joystick.
```

Comment:
should avoid breaking lines in the middle of links
```suggestion
For that, you can use a keyboard for driving your Amiga, or use an
[8Bitdo Sn30](https://shop.8bitdo.com/products/8bitdo-sn30-pro-bluetooth-gamepad) joystick.
```

Reply:
Addressed here: 54f6c0888fb3759d41de4f77f032f7e341ef8393